yarn.lock

```diff
@@ -0,0 +1,3738 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I also noticed the maintainers do not check this file in. We won't do that either. Please remove.

Reply:
The file is back!